devroom-submission.md

```diff
@@ -0,0 +1,44 @@
+# Project details
+## Title of devroom
+
+Embedded, Mobile and Automotive
+
+## Elaborate description of proposed devroom (including possible topics)
+
+For many years the Embedded, Mobile and Automotive devroom was one of
+the most popular devrooms at FOSDEM. After an unfortune break in 2019,
+we are working hard to bring EMA devroom to its finest.
+
+The goal of this devroom is to showcase open source from embedded,
+mobile and automotive areas.
+
+Possible topics include:
+* Hardware hacking
+* FOSS in embedded programming
+* The Linux Kernel
+* FOSS in mobile area
+* FOSS in automotive area
+
+## Why does it fit FOSDEM?
+
+The Embedded, Mobile and Automotive industries heavily depend on
+FOSS. There are so many related open source projects that it is almost
+impossible to know them all. That's why we believe that knowledge
+sharing is a main goal of this devroom.
+
+The devroom used to be a great way of connecting open source
+contributors from all related projects. Resurrecting the devroom will
+attract low level programmers to visit the conference and allow us to
+stimulate embedded community integration.
```

Comment:
I think this paragraph needs to be adjusted, as we're not "resurrecting" the devroom anymore: the resurrection took place last year.